release_notes/Service_Fabric_ReleaseNotes_90CU3.md

```diff
@@ -0,0 +1,83 @@
+* SFRP node types with VMSS Durability of ”Silver”/”Gold” should always have Windows update explicitly disabled to avoid unintended OS restarts due to the Windows updates, which can impact the availability of the workloads or the cluster itself. This can be achieved by setting the "enableAutomaticUpdates": false, in the VMSS OSProfile.Consider enabling Automatic VMSS Image upgrades instead.The deployments will start failing from *09/30/2022* for new clusters, if the WindowsUpdates are not disabled on the VMSS. This validation will be enabled gradually for all clusters.<br>
+For more information see: [VMSS Image Upgrades] (https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-automatic-upgrade)
+
+## Service Fabric Feature and Bug Fixes
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.0.1107.9590<br>Ubuntu 18 - 9.0.1103.1<br>Ubuntu 20 - 9.0.1103.1** | **Bug** | Service Remoting | **Brief Description**: Service Remoting does not honor the MaxConcurrentCalls setting due to which all the incoming service remoting calls are processed simultaneously.<br>**Impact**: MaxConcurrentCalls configuration of Service Remoting is allowing more calls to be processed than the given value.<br>**Fix**: MaxConcurrentCalls will limit the number of incoming service remoting calls to be processed concurrently. The default value of MaxConcurrentCalls setting is changed to 0 which implies that all the incoming service remoting calls would be processed concurrently by the service. Any non-zero value for MaxConcurrentCalls setting will limit the number of incoming service remoting calls being processed concurrently by the service.
+| **Windows - 9.0.1107.9590<br>Ubuntu 18 - 9.0.1103.1<br>Ubuntu 20 - 9.0.1103.1** | **Feature** | Backup Restore Service | **Brief Description**: BackupRestoreService(BRS) in Service Fabric is updated with new events for all backup and restore operations.This feature adds events for each intermediate state during backup and restore operations at partition level. BRS will also fire periodic events about backup and policies at partition and cluster level. These events can be viewed on Service Fabric Explorer.<br> **Documentation Reference**: Read more about it at - [BackupRestoreService events](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-diagnostics-event-generation-operational) 
+
+## Retirement and Deprecation Path Callouts
```

Comment:
Hey, what happened to your plans of separating this section out into a separate documents page and putting up a link here? Any ETA on when you plan to do that?

Reply:
Yes. I was working on it and Zn dependency happened. ETA would be by next release